electroncash/blockchain.py

```diff
@@ -356,7 +356,7 @@ def write(self, data, offset, truncate=True):
 
     def save_header(self, header):
         delta = header.get('block_height') - self.base_height
-        data = bfh(serialize_header(header))
+        data = bitcoin.bfh(serialize_header(header))
```

Comment:
Is this intended to be `bytes.fromhex()` similar to the change above?

Reply:
Yes. I will fix this. 
It is an indirect import of a bfh function defined in util.py that was indirectly import via bitcoin.py.
And it's basically an alias for bytes.fromhex, so the import is not needed.